hibp.xml

```diff
@@ -1,26 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<extension version="3.8" type="plugin" group="user" client="site" order="100" method="upgrade">
-	<name>User - HIBP</name>
+<extension version="3.8" type="plugin" group="user" client="site" method="upgrade">
+	<name>plg_user_hibp</name>
```

Comment:
Shouldn't the language reference in the name element be upper case?

Reply:
Well we could argue this is the name of the plugin that is also a language string :-D I personally more like to write it in lowercase at this place and iirc it is also done so in the core. But as this is nothing technical relevant i'm happy to write it in uppercase too, just let me know what you think would fit best
 :-)